js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -738,6 +739,20 @@ export class MiniAppBridge {
       );
     });
   }
+
+  getHostAppColorTheme() {
+    return new Promise<HostColorScheme>((resolve, reject) => {
+      console.log("BRIDGE: getHostAppColorTheme")
+      return this.executor.exec(
+        'getHostAppColorTheme',
```

Comment:
Suggestion: I feel this can be renamed to "getHostAppThemeColors", to sound more relevant to the feature. But its also ok to keep as is.